eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/Channel.scala

```diff
@@ -1123,9 +1116,12 @@ class Channel(val nodeParams: NodeParams, val wallet: OnChainChannelFunder, val
       } else if (d.commitments.remoteNextCommitInfo.left.toOption.exists(_.nextRemoteCommit.txid == tx.txid)) {
         // counterparty may attempt to spend its last commit tx at any time
         handleRemoteSpentNext(tx, d)
-      } else {
+      } else if (tx.txIn.map(_.outPoint.txid).contains(d.metaCommitments.main.fundingTxId)) {
```

Comment:
I think there is nothing to do actually, because we always put the `WatchFundingSpent` _after_ the `WatchFundingConfirmedTriggered`. The pruning will already have taken place and we will be back in the single commitment scenario. The existing handler for rbfed funding txs in `CLOSING` already handles that case.